src/extensions/default/HealthData/main.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
+ *  
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"), 
+ * to deal in the Software without restriction, including without limitation 
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+ * and/or sell copies of the Software, and to permit persons to whom the 
+ * Software is furnished to do so, subject to the following conditions:
+ *  
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *  
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING 
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+ * DEALINGS IN THE SOFTWARE.
+ * 
+ */
+
+/*jslint vars: true, plusplus: true, devel: true, nomen: true,  regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets*/
+
+define(function (require, exports, module) {
+    "use strict";
+    
+    var AppInit                 = brackets.getModule("utils/AppInit"),
+        Menus                   = brackets.getModule("command/Menus"),
+        CommandManager          = brackets.getModule("command/CommandManager"),
+        Strings                 = brackets.getModule("strings"),
+        Commands                = brackets.getModule("command/Commands"),
+        
+        HealthDataNotification  = require("HealthDataNotification");
+    
+    var menu            = Menus.getMenu(Menus.AppMenuBar.HELP_MENU),
+        healthDataCmdId = "healthData.healthDataStatistics";
+    
+    //handles the command execution for healthData menu item
+    function handleHealthDataStatistics() {
+        HealthDataNotification.handleHealthDataStatistics();
```

Comment:
@peterflynn My understanding of including this kind of stub was only when it is not used anywhere else and we explicitly want to register it. Can you please tell any scenario in which HealthDataManager will not load. Have we faced any such thing in past?